src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -445,34 +326,24 @@ public async Task Export_SimulateMultipleExports_ShouldExportCorrectDelta()
 		public async Task Export_MultipleMetricStreams_ShouldExportMultipleLines()
 		{
 			// Arrange
-			using var meter = new Meter(TestUtils.GetCurrentMethodName(), "0.0.1");
-
 			HttpRequestMessage actualRequestMessage = null!;
-			var mockMessageHandler = SetupHttpMock((HttpRequestMessage r) => actualRequestMessage = r);
+			var mockMessageHandler = SetupHttpMock(r => actualRequestMessage = r);
 
 			var sut = new DynatraceMetricsExporter(null, null, new HttpClient(mockMessageHandler.Object));
 
-			var exporterProxy = new TestMetricsExporterProxy(sut);
-
-			var metricReader = new BaseExportingMetricReader(exporterProxy);
-			using var provider = Sdk.CreateMeterProviderBuilder()
-				.AddMeter(meter.Name)
-				.AddReader(metricReader)
-				.Build();
-
-			var counterA = meter.CreateCounter<int>("counterA");
+			var counterA = _meter.CreateCounter<int>("counterA");
 			counterA.Add(10, _attributes);
 
-			var counterB = meter.CreateCounter<int>("counterB");
+			var counterB = _meter.CreateCounter<int>("counterB");
 			counterB.Add(20, _attributes);
 
-			// Act - Reader will call our exporter
-			metricReader.Collect();
+			// Act
+			_meterProvider.ForceFlush();
+			sut.Export(new Batch<Metric>(_exportedMetrics.ToArray(), _exportedMetrics.Count));
 
 			// Assert
-			var exportedMetrics = exporterProxy.GetExportedMetrics();
-			var pointA = MetricTest.FromMetricPoints(exportedMetrics.First().GetMetricPoints()).First();
-			var pointB = MetricTest.FromMetricPoints(exportedMetrics.Last().GetMetricPoints()).First();
+			var pointA = MetricTest.FromMetricPoints(_exportedMetrics.First().GetMetricPoints()).First();
```

Comment:
Can we assert that there are exactly 2 lines easily?

Reply:
I guess we could also assert 1 line for the other tests, we only assert on `First` most of the time

Reply:
`First` throws an exception if there's no matching element found, so we can get away without checking if there's only 1 datapoint https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.first?view=net-6.0

Reply:
If something were to change and add a third line sneaks in in between `First` and `Last` the test would still pass, right?

Reply:
Added an assertion that there are only two metric points in [b53bf26](https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/pull/31/commits/b53bf26575afc2aed853aa8db2805d0e8e132f65) :slightly_smiling_face: